Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -153,7 +132,6 @@ protected void resetTest(String testName) throws Exception {
         if (!currentTestName.equals(testName)) {
             return;
         }
-        wireMock=null;
```

Comment:
Is there api in wiremockrule to reset mapping? if yes i think we need to add it.